Docker.DotNet.X509/CertificateCredentials.cs

```diff
@@ -18,6 +19,19 @@ public CertificateCredentials(X509Certificate2 clientCertificate)
             _certificate = clientCertificate;
         }
 
+#if (NET45 || NET46)
```

Comment:
Should we open an issue now to fix this when support for netstandard vNext includes crypto?